geode-redis/src/test/java/org/apache/geode/redis/internal/executor/key/ExpireAtExecutorJUnitTest.java

```diff
@@ -34,30 +37,30 @@
 
   @Test
   public void calledWithTooFewCommandArguments_returnsError() {
-    List<byte[]> commandsAsBytesWithTooFewArguments = new ArrayList<>();
-    commandsAsBytesWithTooFewArguments.add("EXPIREAT".getBytes());
-    commandsAsBytesWithTooFewArguments.add("key".getBytes());
-    Command command = new Command(commandsAsBytesWithTooFewArguments);
+    List<byte[]> commandAsBytes = new ArrayList<>();
+    commandAsBytes.add("EXPIREAT".getBytes());
+    commandAsBytes.add("key".getBytes());
+    Command command = new Command(commandAsBytes);
 
-    RedisResponse response =
-        new ExpireAtExecutor().executeCommand(command, mockContext());
+    Throwable thrown = catchThrowable(() -> command.execute(mockContext()));
```

Comment:
Can this be a `assertThatThrownBy` assertion?

Reply:
yeah! good catch.